docs/geedocs/answer/3230807.html

```diff
@@ -42,19 +42,19 @@
 <p>You can customize or extend GEE Portable, or create applications that work with it.</p>
 <p>To help you get started, Google provides two <code>Hello World!</code> files that you can copy and use as templates. These files (<code>Hello_Maps.html</code> for 2D maps and <code>Hello_Earth.html</code> for 3D globes) call the required scripts, load the globe or map, define the <code>.css</code> file, and so on. The files include JSON and, if needed, KML polygons.</p>
 <p>See the resources linked to below to learn how to edit and extend the <code>Hello_Maps.html</code> and <code>Hello_Earth.html</code> files.</p>
-<p><strong>Note</strong>: Although it is possible to configure GEE Portable to display multiple globes or maps at the same time, this isn't recommended or supported. The additional configuration requires you to build the globe or map to reference a specific port number, so you can no longer re-use it for any other ports.</p>
+<p><strong>Note</strong>: Although it is possible to configure GEE Portable to display multiple globes or maps at the same time, this isn't recommended or supported.</p>
```

Comment:
I'm, actually, not sure it even applies the way it's written. I think in Portable Server 5, you'll have to run two separate instances of the server on different ports to serve multiple globes or maps. I don't think you can server them from a single server.

I think I may just remove that paragraph.